rest_framework-stubs/serializers.pyi

```diff
@@ -121,7 +122,8 @@ class Serializer(BaseSerializer):
     def _readable_fields(self) -> List[Field]: ...
     def get_fields(self) -> Dict[str, Field]: ...
     def _read_only_defaults(self) -> Dict[str, Any]: ...
-    def validate(self, attrs):
+    T = TypeVar("T")
+    def validate(self, attrs: T) -> T:
```

Comment:
Yes, let's test this. 

Here's an example: https://github.com/typeddjango/djangorestframework-stubs/blob/master/test-data/typecheck/test_serializers.yml